nearswap/src/lib.rs

```diff
@@ -817,17 +819,19 @@ mod tests {
 
     #[test]
     fn add_liquidity_happy_path() {
-        let (mut ctx, mut c) = init();
+        let ynear_deposit = 3 * NDENOM;
+        let token_deposit = 1 * NDENOM;
+        let ynear_deposit_with_storage = ynear_deposit + NEP21_STORAGE_DEPOSIT;
```

Comment:
`NEP21_STORAGE_DEPOSIT` should be removed.

Reply:
ok

Reply:
this will be updated when add_liquidity function i.e new_model branch. removing this from here but test will fail in this current state.